flink-libraries/flink-table/src/main/scala/org/apache/flink/table/api/queryConfig.scala

```diff
@@ -37,6 +37,13 @@ class BatchQueryConfig private[table] extends QueryConfig
 class StreamQueryConfig private[table] extends QueryConfig {
 
   /**
+    * The non-windowed groupby aggregate update the calculation result according a configuration of
+    * time interval. By default non-windowed groupby aggregate will update calculation result each
+    * row.
+    */
+  private var unboundedAggregateUpdateInterval: Time = Time.milliseconds(0)
```

Comment:
I think the distinction makes sense but doubt that everybody would make the same distinction. I'm fine with `updateInterval` but I also think that we need an additional parameter to specify whether the update interval should be in event-time or processing-time.